extensions/resource-deployment/src/ui/resourceTypePickerDialog.ts

```diff
@@ -18,11 +19,15 @@ const localize = nls.loadMessageBundle();
 
 export class ResourceTypePickerDialog extends DialogBase {
 	private toolRefreshTimestamp: number = 0;
+	private _resourceTypes!: ResourceType[];
 	private _selectedResourceType: ResourceType;
 	private _view!: azdata.ModelView;
 	private _optionsContainer!: azdata.FlexContainer;
 	private _toolsTable!: azdata.TableComponent;
+	private _resourceTagsListView!: azdata.ListViewComponent;
+	private _resourceSearchBox!: azdata.InputBoxComponent;
 	private _cardGroup!: azdata.RadioCardGroupComponent;
+	private _resourceContainer!: azdata.FlexContainer;
```

Comment:
this variable name is not clear enough, I can't tell what is being added to it by its name